src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public abstract class Figure implements Forming {
+    private String name;

```

Comment:
Lets remove this field from class Figure, because if you want to create some figure, for example Circle, you can set name - "Rectange" and your code compile properly. You may set name for figure inside draw() method directly for each figure.